proposals/function-pointers.md

```diff
@@ -243,6 +243,26 @@ The better function member specification will be changed to include the followin
 
 This means that it is possible to overload on `void*` and a `delegate*` and still sensibly use the address-of operator.
 
+## Metadata representation of `in`, `out`, and `ref readonly` parameters and return types
+
+Function pointer signatures have no parameter flags location, so we must encode whether parameters and the return type are `in`, `out`, or `ref readonly` by using modreqs.
+
+### `in`
+
+We reuse `System.Runtime.InteropServices.InAttribute`, applied as a `modreq` to the ref specifier on a parameter or return type, to mean the following:
```

Comment:
Since `in` maps to `IsReadOnlyAttribute` in regular signatures, would it make more sense to use `IsReadOnlyAttribute` here as well?

Reply:
I wanted to keep using `InAttribute` because we don't emit `IsReadOnlyAttribute` as a modreq anywhere today, but we do for `InAttribute`.

Reply:
I'm not opposed to using `IsReadOnlyAttribute` instead if we so desire.

Reply:
ok